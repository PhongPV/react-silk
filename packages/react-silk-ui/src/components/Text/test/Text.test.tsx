import * as React from 'react'
import TestRenderer from 'react-test-renderer'

import { Text } from '../index'

describe('<Text />', () => {
  test('renders without errors', () => {
    const text = TestRenderer.create(<Text />)
    expect(text.toJSON()).toBeTruthy()
  })

  test('match snapshot', () => {
    const text = TestRenderer.create(<Text />)
    expect(text.toJSON()).toMatchSnapshot()
  })
})
