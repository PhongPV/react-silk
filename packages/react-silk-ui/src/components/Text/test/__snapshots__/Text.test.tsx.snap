// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> match snapshot 1`] = `
<div
  dir="auto"
  style={
    Object {
      "borderBottomWidth": "0px",
      "borderLeftWidth": "0px",
      "borderRightWidth": "0px",
      "borderTopWidth": "0px",
      "boxSizing": "border-box",
      "color": "inherit",
      "display": "inline",
      "fontFamily": "system-ui, -apple-system, BlinkMacSystemFont, \\"Segoe UI\\", Roboto, Ubuntu, \\"Helvetica Neue\\", sans-serif",
      "fontSize": "16px",
      "fontStyle": "inherit",
      "fontVariant": "inherit",
      "fontWeight": "inherit",
      "lineHeight": "inherit",
      "marginBottom": "0px",
      "marginLeft": "0px",
      "marginRight": "0px",
      "marginTop": "0px",
      "paddingBottom": "0px",
      "paddingLeft": "0px",
      "paddingRight": "0px",
      "paddingTop": "0px",
      "textDecoration": "none",
      "whiteSpace": "pre-wrap",
      "wordWrap": "break-word",
    }
  }
/>
`;
